Topics/Development_Process/Standards_For_Best_Practices.md

```diff
@@ -0,0 +1,47 @@
+# Utilizing Standards to Adopt Best-Practices in Software Engineering
+
+## Table of Contents
+
+### [What Is a Standard?](#what-is-a-standard-1)
+
+### [Why Are Standards Important?](#why-are-standards-important-1)
+
+### [How Are Standards Created?](#how-are-standards-created-1)
+Aligned with the guidelines set forth by the Digital Governance Standards Institute (DGSI), a technical committee comprising of subject matter experts is established prior to the creation of a standard, particularly if there is not already a committee addressing the specific topic. These committees operate by consensus to ensure inclusivity and equal participation, access, and opportunity among interested parties.
+
+
+The technical committees have the responsibility of developing, approving, interpreting, and periodically reviewing standards within their respective subjects, ensuring currency and relevance.
+
+Any individual or organization, known as stakeholders, can suggest the creation, modification, or removal of a standard. Upon approval of a new standard, the technical committee initiates the standard development process, beginning with drafting the initial version. 
+
+Following the completion of the first draft, a public review period ensues. The public then has the opportunity to comment, provide feedback, and suggest improvements. 
+
+Following thorough consideration of public comments and necessary revisions, the committee reaches a consensus on the final version of the standard. Subsequently, it is then submitted for approval by relevant governing bodies or organizations before being officially published. 
+
+Overall, the standardization process ensures consistency, quality, and interoperability, benefiting stakeholders and facilitating innovation and growth in the respective field.
+
+### [Why is This Relevant to CSC301 Students?](#why-is-this-relevant-to-csc301-students-1)
+Understanding and learning about standards is crucial for CSC301 students for several reasons:
+* Ensuring quality: Standards provide frameworks and guidelines that are easy to follow and ensure quality, reliability, and interoperability. 
```

Comment:
It might be useful here to give an example. For example, "A CSC301 student might find that the standard X would help with ensuring SWE task Y is reliable"

Reply:
Will do